.github/workflows/conformance-datapath.yaml

```diff
@@ -180,6 +182,12 @@ jobs:
           docker cp $cid:/usr/local/bin/cilium ./cilium-cli
           docker rm $cid
 
+      # For `go test -c` in the verifier step below.
```

Comment:
I think this is not needed, as you compile in the LVH VM, right?

Reply:
Ah nice, the LVM VM has `go` as well. Is this guaranteed to be the same version as in `GO_VERSION`? Otherwise it's one repo more for me to keep track of to bump Go versions :smile: